stock_picking_invoice_link/migrations/15.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2023 ForgeFlow, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import logging
+
+
+def migrate(cr, version):
+    if not version:
+        return
+    logger = logging.getLogger(__name__)
+    logger.info("Creating column: delivery_count for table account_move")
+    cr.execute(
+        """
+    ALTER TABLE account_move
+        ADD COLUMN IF NOT EXISTS delivery_count INTEGER;
+    """
+    )
+    logger.info("Populating values for column delivery_count in table account_move")
+    cr.execute(
```

Comment:
Please use `openupgrade.logged_query`. It's very important to log the number of changes done for traceability.